AK/Noncopyable.h

```diff
@@ -29,9 +29,9 @@
 #define AK_MAKE_NONCOPYABLE(c) \
 private:                       \
```

Comment:
Is the `private:` even needed now that these are `= delete`?

Reply:
I didn't check whether any of the ~80 usages depends on that. My main motivation was to get rid of "double-semicolon" warnings I sometimes see with specific compilers/options. :)